database.py

```diff
@@ -119,26 +135,8 @@ def setup(self):
             ('Red', 'R'),
             ('Green', 'G')
         """)
-        self.execute('DELETE FROM rarity')
-        self.execute("""INSERT INTO rarity (name) VALUES
-            ('Common'),
-            ('Uncommon'),
-            ('Rare'),
-            ('Mythic Rare'),
-            ('Special')
-        """)
 
-    # Drop All Tables, so we can reinit
-    def droptables(self):
-        self.execute('DROP TABLE IF EXISTS card')
-        self.execute('DROP TABLE IF EXISTS card_alias')
-        self.execute('DROP TABLE IF EXISTS card_color')
-        self.execute('DROP TABLE IF EXISTS card_color_identity')
-        self.execute('DROP TABLE IF EXISTS card_name')
-        self.execute('DROP TABLE IF EXISTS card_subtype')
-        self.execute('DROP TABLE IF EXISTS card_supertype')
-        self.execute('DROP TABLE IF EXISTS card_type')
-        self.execute('DROP TABLE IF EXISTS color')
-        self.execute('DROP TABLE IF EXISTS rarity')
-        self.execute('DROP TABLE IF EXISTS version')
-        self.execute('DROP TABLE IF EXISTS db_version')
+    # Drop the database so we can recreate it.
+    def delete(self):
+        os.remove(configuration.get('database'))
```

Comment:
This won't work on Windows.  You've still got the file handle open from earlier.